src/NeoSharp.Application/Controllers/PromptBlockchainController.cs

```diff
@@ -16,6 +17,7 @@ public class PromptBlockchainController : IPromptController
         private readonly IBlockPool _blockPool;
         private readonly ITransactionPool _transactionPool;
         private readonly IBlockchain _blockchain;
+        private readonly IBlockchainContext _blockchainContext;
```

Comment:
I assume this is the code for the issue describing too many params in Blockchain class. Could we keep it separately?

Reply:
Issue #428 added